### PR TITLE
nvme: add support for lba status log page

### PR DESCRIPTION
**Pull request by ganaidu707**
This log page is used to provide information about
subsequent actions the host may take to discover which
logical blocks, in namespaces that are attached to
the controller, may no longer be recoverable when read.
For more details see NVM Express 1.4 Spec. Section
5.14.1.14("LBA Status Information (Log Identifier 0Eh)")

Signed-off-by: Gollu Appalanaidu <anaidu.gollu@samsung.com>
Co-Authored-By: Karthik Balan <karthik.b82@samsung.com>

### CODE REVIEW

File: nvme.c [hunk 1]
```diff
@@ -1097,6 +1097,74 @@ static int get_endurance_event_agg_log(int argc, char **argv,
...
+
+	err = fd = parse_and_open(argc, argv, desc, opts);
+	if (fd < 0)
+		goto ret;
+
+	err = flags = validate_output_format(cfg.output_format);
+	if (flags < 0)
+		goto close_fd;
+
+	err = nvme_lba_status_log(fd, &lslplen, cfg.rae, sizeof(__u32));
```
Comment (keithbusch) [inline nvme.c:1136]: You should use 'true' for RAE when just getting the length of the log. The user's RAE selection should only apply to the final log request.

File: nvme.c [hunk 2]
```diff
@@ -1097,6 +1097,74 @@ static int get_endurance_event_agg_log(int argc, char **argv,
...
+		perror("lba status log page");
+		goto close_fd;
+	} else if (err) {
+		nvme_show_status(err);
+		goto close_fd;
+	}
+
+	lab_status = calloc(lslplen, 1);
+	if (!lab_status) {
+		fprintf(stderr, "could not alloc buffer for lba status " \
```
Comment (keithbusch) [inline nvme.c:1147]: Please use `perror` insetead of fprintf for notifying system errors that already set errno.

File: nvme.c (hunk 1 above)
Comment (ganaidu707) [reply nvme.c]: yeah, correct, taken care.

File: nvme.c (hunk 2 above)
Comment (ganaidu707) [reply nvme.c]: changed,

**Pull request merged**